circleci/Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN set -ex \
     xz-utils
 
 # Golang
+# The format of the GO args shouldn't be changed as the update-go invoke task relies on it
```

Comment:
```suggestion
# If you want to change the 2 below lines, please use the update-go invoke task
```
I suggest this because the other way around could be misleading, wdyt?